### PR TITLE
Crypto account creation check

### PR DESCRIPTION
**Pull request by vimalan-deriv**
In this test case we would be creating all the available cryptocurrency in deriv and check for the confirmation message and also details displayed in the manage account page

### CODE REVIEW

File: cypress/e2e/wip/Crypto/crypto_account_check.cy.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  cy.get('body').then(($body) => {
+    if ($body.find('.notification--warning').length) {
+      cy.get('.notification--warning')
+        .should('exist')
+        .each(($element) => {
+          cy.wrap($element).find('.notification__close-button').should('be.visible').click({ force: true });
+        });
+    }
+  });
+});
```
Comment (abuzar-deriv) [inline cypress/e2e/wip/Crypto/crypto_account_check.cy.js:14]: We already have a common command for this, `c_closeNotificationHeader()`. Can you please replace use that instead and remove this?

File: cypress/e2e/wip/Crypto/crypto_account_check.cy.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        // Element is not disabled
+        cy.closenotificationbanner();
+        cy.findByText(crypto).click();
+        cryptoconfig.elements.cryptoaddaccount().should('be.visible').click();
+        cy.findByText('Success!')
+        cy.findByText('Make a deposit now to start trading.')
+        cryptoconfig.elements.maybelater().should('be.visible').click();
+      }})
+    return crypto;
+  };
```
Comment (abuzar-deriv) [inline cypress/e2e/wip/Crypto/crypto_account_check.cy.js:41]: Function should be out of describe block.

File: cypress/e2e/wip/Crypto/crypto_account_check.cy.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  it('should be able to create crypto account from Traders Hub.', () => {
+    cy.wait(1000).closenotificationbanner();
```
Comment (abuzar-deriv) [inline cypress/e2e/wip/Crypto/crypto_account_check.cy.js:43]: Please remove this dynamic wait and wait for an element instead.

File: cypress/e2e/wip/Crypto/crypto_account_check.cy.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    const cryptocurrencies = ["Bitcoin", "Ethereum", "Litecoin", "Tether TRC20", "USD Coin"];
+    const currency_code = ["BTC", "ETH", "LTC", "tUSDT", "USDC"];
```
Comment (abuzar-deriv) [inline cypress/e2e/wip/Crypto/crypto_account_check.cy.js:45]: Please remove semi-colons everywhere.

File: cypress/e2e/wip/Crypto/pageobject/common.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+class cryptoconfig {
+    elements = {
+        currencyswitcher: () => cy.findByTestId('dt_currency-switcher__arrow'),
+        manageaccount: () => cy.get('[class*="dc-btn dc-btn--secondary dc-btn__large block-button"]'),
+        closemanageaccount: () => cy.get('[class*="dc-modal-header__close"]'),
+        cryptoaddaccount: () => cy.get('[class*="dc-btn dc-btn__effect dc-btn--primary dc-btn__large"]'),
```
Comment (abuzar-deriv) [inline cypress/e2e/wip/Crypto/pageobject/common.js:6]: Were cypress testing library locators not available for these? Can you please check with playwright generator. Please ignore if they were not available.

File: cypress/e2e/wip/Crypto/pageobject/common.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+        maybelater: () => cy.findByRole('button', { name: 'Maybe later' }),
+        closenotification: () => cy.get('.notification__close-button')
```
Comment (abuzar-deriv) [inline cypress/e2e/wip/Crypto/pageobject/common.js:8]: Same as above for this.

**Review by abuzar-deriv — CHANGES_REQUESTED** [review]

File: cypress/e2e/wip/Crypto/crypto_account_check.cy.js (hunk 1 above)
Comment (vimalan-deriv) [reply cypress/e2e/wip/Crypto/crypto_account_check.cy.js:14]: I have resolved this issue, please help to check

**Pull request closed without merging**